### DISCUSSION TITLE
Dịch "chain rule"

### DISCUSSION THREAD

**Discussion by cuongvng** [d2l-ai/d2l-vi#2904] [Terminology]
Mọi người thấy nên dịch là **quy tắc dây chuyền/chuỗi** hay **đạo hàm hàm hợp**? 
* quy tắc dây chuyền 👍 
* quy tắc chuỗi ❤️ 
* đạo hàm hàm hợp 😄

cc: @lkhphuc @minhduc0711 @rootonchair @quangnhat185 @davidnvq @trGiang99

**Comment by quangvnai** [comment]
quy tắc dây chuyền thấy rất phổ biến bên Toán đó Cường ;)

**Reply by rootonchair** [reply]
Em lại được dạy là quy tắc mắt xích @@
